website/cypress/support/e2e.ts

```diff
@@ -1 +0,0 @@
-import '@cypress/code-coverage/support';
```

Comment:
this is why the test coverage is no longer there

Reply:
I put it back.